src/include/verbose.h

```diff
@@ -28,6 +28,7 @@
     gen(WT_VERB_EVICT_STUCK)           \
     gen(WT_VERB_EXTENSION)             \
     gen(WT_VERB_FILEOPS)               \
+    gen(WT_VERB_GEN)                   \
```

Comment:
I would be happy with the full name since we have used the full name for "all" the others.

Reply:
Good point, fixed! 396ab71